File: app/src/main/java/com/jedev/leflix/helper/DateCustom.java
package com.jedev.leflix.helper;

import android.util.Base64;

import java.text.SimpleDateFormat;

public class DateCustom {


    public static String dataAtual() {

        long data = System.currentTimeMillis();
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd/MM/yyyy");
        String dataString = simpleDateFormat.format(data);
        return dataString;

    }

    public static class Base64Custom {


        public static String codificarBase64(String texto){
            return Base64.encodeToString(texto.getBytes(), Base64.DEFAULT ).replaceAll("(\\n|\\r)","");
        }

        public static String decodificarBase64(String textoCodificado){
            return new String( Base64.decode(textoCodificado, Base64.DEFAULT) );
        }

    }
}

File: app/src/main/java/com/jedev/leflix/utils/DateCustom.kt
package com.jedev.leflix.utils

import android.util.Base64

import java.text.SimpleDateFormat

object DateCustom {

    object Base64Custom {

        fun codificarBase64(texto: String): String {
            return Base64.encodeToString(texto.toByteArray(), Base64.DEFAULT).replace("([\\n\\r])".toRegex(), "")
        }

        fun decodificarBase64(textoCodificado: String): String {
            return String(Base64.decode(textoCodificado, Base64.DEFAULT))
        }

        fun dataAtual(): String {
            val data = System.currentTimeMillis()
            val simpleDateFormat = SimpleDateFormat("dd/MM/yyyy")
            return simpleDateFormat.format(data)
        }
    }
}
